cloud/linode/linode.py

```diff
@@ -446,6 +463,8 @@ def main():
             api_key = dict(),
             name = dict(type='str'),
             plan = dict(type='int'),
+            stack_script_id = dict(type='int'),
+            stack_script_udf_responses = dict(type='str'),
```

Comment:
you might want to make it type=json as people tend to arbitrarily either assign a JSON string or a data structure they expect to become JSON.